hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestMergeIntoTable.scala

```diff
@@ -261,7 +262,8 @@ class TestMergeIntoTable extends HoodieSparkSqlTestBase with ScalaAssertionSuppo
   }
 
   test("Test MergeInto for MOR table ") {
-    withRecordType()(withTempDir {tmp =>
+    spark.sql(s"set ${MERGE_SMALL_FILE_GROUP_CANDIDATES_LIMIT.key} = 0")
+    withRecordType()(withTempDir { tmp =>
```

Comment:
Why this change

Reply:
This is to ensure that for MOR table, log files are written.  Otherwise, the MOR table generated by the test may not contain log files, which is not different than COW.

Reply:
Got it, is it related with this change?

Reply:
Yes, I'd like to make sure that my changes do not break MERGE INTO on MOR tables.